docs/dashboards/chart-builder.md

```diff
@@ -229,38 +229,87 @@ a comparison of stacked and non-stacked bar charts.
+You can style any series as a heatbar, but it's especially suitable for events and logs.  
 
-![Heatbars example](../images/dashboards/heatbar-example-alerts.png)
+Here you can see a time-series chart which shows the total number of logs and, above it, a heatbar which tracks only errors among them.
 
-Let's say you wanted to track a certain subset of your logs, for example logs which contain the keyword `"Usage"` and additionally, you wanted to show *Usage* errors in a heatbar.
+![Heatbars example](../images/dashboards/heatbar-1-example-usage-logs-with-errors.png)
 
-First, you would add a `Logs` data series, which you would filter by query `"Usage"`.
+A chart can have multiple series configured as heatbars, like in the following example:
 
-![Creating chart with a heatbar](../images/dashboards/heatbar-creating-1.png)
+![More than one Heatbar](../images/dashboards/heatbar-1.1-more-than-one-hb-series.png)
 
-So far so good, we have our `Usage logs`.
+If all of the series are heatbars the main time-series chart is not even shown.  
+For example, the following chart is configured to show a single heatbar series, `CPU User` [grouped by](#group-by) `OS Host`:
 
-![Usage logs](../images/dashboards/heatbar-creating-2.png)
+![Just heatbars](../images/dashboards/heatbar-1.2-only-cpu-user-by-host.png)
 
-Next, you would add another Logs data series with the same `"Usage"` query, but you would additionally filter logs to only those with `severity: error`.
+The chart shows six heatbars because the `CPU User` metric was reported by six different `OS Hosts` during the observed time range.
 
-![Severity error as a heatbar](../images/dashboards/heatbar-creating-3-severity-error.png)
+### Creating Heatbars
 
-Finally, in the `Style` section of our second data series, you would change its type to `heatbar`.
+Let's build the [Usage logs with errors](#heatbar) component together.  
 
-![Changing data series type](../images/dashboards/heatbar-creating-4-ds-style-type.png)
+Since the total number of logs is not particularly interesting, we'll monitor a certain subset of logs, for example logs which contain the word `Usage`.  
+We'll have our time-series chart tracking all *Usage* logs and additionally, we'll use a heatbar to track only errors, like we've seen in the first Heatbar example above.  
 
-And just like that, we have our *Usage chart* with a heatbar of *Usage errors*.
+In your dashboard, click on the `Add component` button on the top of the screen, which looks like this:
+
+![Add component](../images/dashboards/heatbar-2-add-component.png)
+
+In the first dialog choose `Time Series Chart`:
+
+![Time-series chart](../images/dashboards/heatbar-2.1-time-series-chart.png)
+
+Then choose `Logs` data series:
+
+![Logs data series](../images/dashboards/heatbar-2.2-logs-data-series.png)
+
+If you're creating your first chart, the initial result may look intimidating at the first glance:
```

Comment:
If you're creating your first chart, the initial result may look intimidating at the first glance:
=>
If you're creating your first chart, the initial result may look intimidating at first glance: